src/chaincode/dex/transferUnclaimedFunds.ts

```diff
@@ -99,10 +99,11 @@ export async function transferUnclaimedFunds(
   );
 
   // Transfer funds to secure wallet
+  const transferTo = asValidUserAlias(dto.secureWallet);
```

Comment:
this should be changed to `resolveUserAlias(ctx, dto.secureWallet);`

we want to find the user alias from the user ref. The current code just does a "safe" casting with some validation